src/main/java/co/zecko/retailer/client/httpClient/CartClient.java

```diff
@@ -0,0 +1,120 @@
+        Map<String, String> headers = getBaseHeaders(zeckoAccessToken);
+        String routeUri = "/delete";
+        String url = getUrl(BASE_URI + routeUri);
+
+        return patch(url, cartActionRequest, headers, new TypeReference<>() {});
+    }
+
+
+    public CartData update(String legacyDraftOrderId, CartUpdateRequest cartUpdateRequest)
+            throws BaseException, IOException, InterruptedException{
+        zeckoAccessToken = getZeckoAccessToken(zeckoAccessToken);
+        Map<String, String> headers = getBaseHeaders(zeckoAccessToken);
+        String routeUri = String.format("/%s/update", legacyDraftOrderId);
+        String url = getUrl(BASE_URI + routeUri);
+
+        return patch(url, cartUpdateRequest, headers, new TypeReference<>() {});
+    }
+
+    public CartData addDiscount(String legacyDraftOrderId, CartDiscountRequest cartDiscountRequest)
+            throws BaseException, IOException, InterruptedException{
+        zeckoAccessToken = getZeckoAccessToken(zeckoAccessToken);
+        Map<String, String> headers = getBaseHeaders(zeckoAccessToken);
+        String routeUri = String.format("/%s/add-discount", legacyDraftOrderId);
+        String url = getUrl(BASE_URI + routeUri);
+
+        return patch(url, cartDiscountRequest, headers, new TypeReference<>() {});
+    }
+
+    public CartData removeDiscount(String legacyDraftOrderId, CartDiscountRequest cartDiscountRequest)
+            throws BaseException, IOException, InterruptedException{
+        zeckoAccessToken = getZeckoAccessToken(zeckoAccessToken);
+        Map<String, String> headers = getBaseHeaders(zeckoAccessToken);
+        String routeUri = String.format("/%s/remove-discount", legacyDraftOrderId);
+        String url = getUrl(BASE_URI + routeUri);
+
+        return patch(url, cartDiscountRequest, headers, new TypeReference<>() {});
+    }
+
+    public OrderData complete(String legacyDraftOrderId, CartCompleteRequest cartCompleteRequest)
+            throws BaseException, IOException, InterruptedException{
+        zeckoAccessToken = getZeckoAccessToken(zeckoAccessToken);
+        Map<String, String> headers = getBaseHeaders(zeckoAccessToken);
+        String routeUri = String.format("/%s/complete", legacyDraftOrderId);
+        String url = getUrl(BASE_URI + routeUri);
+
+        return post(url, cartCompleteRequest, headers, new TypeReference<>() {});
+    }
+
+    public DraftOrderDeleteResponseWrapper delete(String clientCustomerId)
+            throws BaseException, IOException, InterruptedException{
+        zeckoAccessToken = getZeckoAccessToken(zeckoAccessToken);
+        Map<String, String> headers = getBaseHeaders(zeckoAccessToken);
+        Map<String, String> queryParams = new HashMap<>();
+        String routeUri = "";
+        String url = getUrl(BASE_URI + routeUri);
+
+        if (StringUtils.isEmpty(clientCustomerId)) {
+            String message = "Customer ID can not be empty";
```

Comment:
Let's make messages consistent across all APIs and client:
`Missing required parameter clientCustomerId`

Reply:
done